magento/magento2
# Resolved issue if checkbox custom option is optional

Resolved issue if checkbox custom option is optional


### Description (*)

On frontend, unable to add to cart if product's custom option is checkbox and optional.



### Manual testing scenarios (*)

1. Have a Magento v2.3 clean installation.
2. Create a product with the checkbox custom option and keep that option as optional.
3. **Before:** Add to cart without selecting any option and you will get `The product's required option(s) weren't entered. Make sure the options are entered and try again.` error.
**After:** This error disappears in this fix.

### Contribution checklist (*)
